### ISSUE TITLE
Would be helpful to be able to set/adjust the date when checking in

### ISSUE THREAD

**Issue by jsocolof** [CorWatts/fasterscale#207]
If a user wants to do a check-in a prior day, would be nice to be able to set the date for the check-in. E.g. when wanting to do the check-in the next morning for the previous day.

**Comment by CorWatts** [comment]
Hey @jsocolof. Thanks for the suggestion. This feels somewhat related to (though not the same as) ticket #135 .

**Comment by jsocolof** [comment]
Yes, @CorWatts - related, and both would be nice to be able to do.
